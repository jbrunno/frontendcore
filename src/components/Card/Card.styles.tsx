import { styled } from '@mui/material/styles';
import { Card as MuiCard } from '@mui/material';

export const CardStyled = styled(MuiCard)`
  background-color: ${({ theme }) => theme.palette.background.paper};
`;
