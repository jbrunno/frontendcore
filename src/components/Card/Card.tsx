import React from 'react';
import { CardProps as MuiCardProps } from '@mui/material';
import { CardStyled } from './Card.styles';

export type CardProps = MuiCardProps;

export function Card(props: CardProps) {
  return <CardStyled {...props} />;
}

export default Card;
