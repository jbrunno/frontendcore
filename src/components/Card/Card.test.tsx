import React from 'react';
import { render } from '@test-utils/component-testing';
import { Card } from './Card';

test('renders the Card', () => {
  const { container } = render(<Card />);
  expect(container).toMatchSnapshot();
});
