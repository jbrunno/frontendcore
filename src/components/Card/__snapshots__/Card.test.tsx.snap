// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the Card 1`] = `
.emotion-1 {
  background-color: #fff;
  color: #3B3B41DE;
  -webkit-transition: box-shadow 300ms cubic-bezier(0.4, 0, 0.2, 1) 0ms;
  transition: box-shadow 300ms cubic-bezier(0.4, 0, 0.2, 1) 0ms;
  border-radius: 4px;
  box-shadow: 0px 2px 1px -1px rgba(0,0,0,0.2),0px 1px 1px 0px rgba(0,0,0,0.14),0px 1px 3px 0px rgba(0,0,0,0.12);
  overflow: hidden;
  background-color: #fff;
}

<div>
  <div
    class="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 MuiCard-root emotion-0 emotion-1"
  />
</div>
`;
